common/src/main/resources/less/style.less

```diff
@@ -1027,3 +1051,74 @@ html {
 .observer-notes {
   padding: 1em;
 }
+
+.ui.segment.explore-magnitudes-container {
+  padding: 0;
+  display: inline-block;
+}
+
+// https://css-tricks.com/responsive-data-tables/
+.tile-xs-width .explore-magnitudes-container .ui.ui.ui.ui.table:not(.unstackable) {
+  tfoot {
+    display: block;
+  }
+
+  td {
+    /* Behave  like a "row" */
+    position: relative;
+    padding-left: 40%;
+    padding-right: 0.5em;
+  }
+
+  td::before {
+    /* Now like a table header */
+    position: absolute;
+
+    /* Top/left values mimic padding */
+    top: 6px;
+    left: 6px;
+    width: 45%;
+    padding-left: 0.25em;
+    white-space: nowrap;
+  }
+
+  td:first-child::before {
+    font-weight: normal;
+  }
+
+  // It is a bit unfortunate that we have to hardcode the column names here
+  td:nth-of-type(1)::before {
```

Comment:
Yeah, this is a bit of a bummer. But, it's amazing you could make the tables responsive at all. :)